pkg/engine/utils/utils.go

```diff
@@ -12,29 +12,6 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
-//RuleType defines the type for rule
-type RuleType int
-
-const (
-	//Mutation type for mutation rule
-	Mutation RuleType = iota
-	//Validation type for validation rule
-	Validation
-	//Generation type for generation rule
-	Generation
-	// ImageVerify type for image verification
-	ImageVerify
-)
-
-func (ri RuleType) String() string {
-	return [...]string{
-		"Mutation",
-		"Validation",
-		"Generation",
```

Comment:
Looks like we didn't add the new rule type for `ImageVerify` when the rule was added.

Reply:
Yes, i asked myself the same thing when refactoring the code (I'm not sure if `All` is used at all TBH)